scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -673,7 +664,17 @@
             ;; Need to lock on fenzo when accessing scheduleOnce because scheduleOnce and
             ;; task assigner can not be called at the same time.
             ;; task assigner may be called when reconciling
+            to-unassign (util/set-atom! unassign-task-set #{})
```

Comment:
Why not use [`reset-vals!`](https://clojuredocs.org/clojure.core/reset-vals!) instead of `util/set-atom!`?

Reply:
@scrosby Thoughts on this?